src/front-end/components/log.jsx

```diff
@@ -0,0 +1,45 @@
+import { currentObject, header } from "../index.js";
+import {SettingsMenu} from "./settings.jsx";
+import { router } from "../state/router.js";
+
+// JSX Engine Import
+/* eslint-disable */
+/** @jsx createElement */
+/** @jsxFrag createFragment */
+import { createElement, createFragment } from "../jsxEngine.js";
+/* eslint-enable */
+
+let template = <template>
+	<link type="text/css" rel="stylesheet" href="log.css" />
```

Comment:
You should remove this line since the client will waste time making a network request to get this file, which is just empty.
You should also delete `src/front-end/components/log.css` so nobody tries to use it.

Reply:
This is an incomplete file that he is working on for the next commit. I would ignore it for now since he will be putting stuff in them over the next couple of days but yeah thanks for pointing it out!

Reply:
I think it's worth it to do it for now, since it's always possible things will change and this may end up forgotten about.

Reply:
honestly i barely wrote code for it so its ok to remove, i agree it professionally shouldnt be there